### ISSUE TITLE
[cms] data call in HTML vizzes

### ISSUE THREAD

**Issue by davelandry** [Datawheel/canon#934]

**Comment by jhmullen** [comment]
@davelandry Can you remember the purpose for this? Would the datacall have a dataFormat, where you could then return the link you construct?

**Comment by davelandry** [comment]
@jhmullen exactly. Wanting to construct the HTML based on the return of a data call. Currently, you would need to make a generator to render the HTML you need, so this would just be a small efficiency to lighten the initial generator payload and offload that logic to the client.
